proto-public/pbmulticluster/v2beta1/computed_exported_services.proto

```diff
@@ -11,17 +11,17 @@ import "pbresource/resource.proto";
 message ComputedExportedServices {
   option (hashicorp.consul.resource.spec) = {scope: SCOPE_PARTITION};
 
-  repeated ComputedExportedService consumers = 1;
+  repeated ComputedExportedService services = 1;
```

Comment:
Each item in this list is actually a "service" which then itself _has_ consumers.